### DISCUSSION TITLE
Latest OpenAI API Models

### DISCUSSION THREAD

**Discussion by sakib1486** [danny-avila/LibreChat#2232] [Help Wanted]
Hi, 
just a simple and basic question. I have the older version of LibreChat that I am using only with OpenAI. I am not using the latest one with the Assistants API enabled. But what I wanted to know is if I can use the latest OpenAI models like `gpt-4-0125-preview` in the older version. Is only updating the `.env` enough with models name?

What I have done is this, but it there is no way to verify if the API is using the latest model that I want.
![Screenshot 2024-03-28 141545](https://github.com/danny-avila/LibreChat/assets/58849295/53e4546c-176d-4e10-86f2-72f16b76ac2e)

The `gpt-4-turbo-preview` if working properly, should point to the latest `gpt-4-0125-preview model released by OpenAI.

**Comment by danny-avila** [comment]
It may work and you will know by the response. However, depending on how old, LibreChat may not know the model's token context window.

**Reply by danny-avila** [reply]
the config file will come in handy when you can manually set the context windows yourself, but that will be on latest and not yet implemented atm

**Reply by sakib1486** [reply]
I am composing the container with the `latest:image` though. From the response it seems it is using the latest model of 2023. Should be good for now.
